include/cosim/system_structure.hpp

```diff
@@ -213,8 +213,47 @@ class system_structure
         full_variable_name target;
     };
 
+    /// Information about a powerbond connection. For use with the ecco algorithm only.
+    struct power_bond
+    {
+        /// The u_a variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name u_a{"", ""};
+
+        /// The u_b variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name u_b{"", ""};
+
+        /// The y_a variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name y_a{"", ""};
+
+        /// The y_b variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name y_b{"", ""};
```

Comment:
These variable names are not very descriptive. I suggest replacing `u` with `input` and `y` with `output`, and possibly swapping the order of the second and third declaration.

Reply:
Good idea, `u` and `y` were taken from the paper of course :). Replaced `u` with `input` and `y` with `output` everywhere.